.github/workflows/android.yml

```diff
@@ -31,6 +31,16 @@ jobs:
         git push origin main
         cd ..
         bash -x subsurface/scripts/get-or-create-build-nr.sh ${{ github.sha }}
+        cp nightly-builds/latest-subsurface-buildnumber subsurface
```

Comment:
What is the reason that we are storing this information in a file inside the action, and not loading it into an environment variable or supplying it as a command line parameter to the scripts that consume it?

Reply:
Because this is real during build time by `scripts/get-version`
That allows us to use the same mechanism for builds in GitHub or from that files (Launchpad and Copr)